packages/@aws-cdk/core/lib/cfn-resource.ts

```diff
@@ -438,6 +439,16 @@ export interface ICfnResourceOptions {
    */
   version?: string;
 
+  /**
+   * The description of this resource.
+   * Used for informational purposes only,
+   * is not processed in any way
+   * (and stays with the CloudFormation template,
+   * is not passed to the udnerlying resource,
```

Comment:
what's with the line length? is it intentionally shorter? i feel like some of it flows better if it's on the same line :)

Reply:
Done!

```suggestion
  /**
   * The description of this resource.
   * Used for informational purposes only, is not processed in any way
   * (and stays with the CloudFormation template, is not passed to the underlying resource,
```